functional_tests/testdata/expected_eks_values/expected_cluster_receiver.yaml

```diff
@@ -3916,6 +3976,27 @@ resourceMetrics:
                       value:
                         stringValue: k8scluster
                   timeUnixNano: "1000000"
+                - asInt: "1"
+                  attributes:
+                    - key: k8s.cluster.name
+                      value:
+                        stringValue: rotel-eks
+                    - key: k8s.namespace.name
+                      value:
+                        stringValue: default
+                    - key: k8s.replicaset.name
+                      value:
+                        stringValue: dotnet-test-57564b7dc9
```

Comment:
Is this id always the same?

Reply:
Same answer as above with one note. This value will always be prefixed with "dotnet-test-" and a random unique value will be appended to it.